src/components/DayPickerRangeController.jsx

```diff
@@ -523,7 +523,10 @@ export default class DayPickerRangeController extends React.PureComponent {
       disabled,
     } = this.props;
 
-    if (e) e.preventDefault();
+    if (e) {
+      e.preventDefault();
+      e.stopPropagation();
```

Comment:
i'm pretty uncomfortable with ever stopping event propagation.

can you add a regression test?

Reply:
Eyyy after update the rest of dependencies of my project it works properly!!
In any case, thanks a lot for the time spent on that.
I think it's a tricky case ;)

Reply:
I was thinking that it is not your fault... it’s caused by material ui library https://material-ui.com/.

I’m going to explain to myself.

Currently I’m using react-dates into a modal from material ui, and after update from version 1 to 3, some events it wasn’t controlled from them side, but it’s good to you, to know that this error could will be happen into a modal.

This error only happen when I set withPortal option or withFullScreenPortal.

Reply:
After all, I had to do a tricky fix, because using DateRangePicker with prop withModal it's impossible capture the events of the calendar.
For this reason I had to edit the styles like a modal and capture the event on click background to close the calendar and clean the input values.
It's to tricky... but it's the solution that works to me... If you it will apply this fix, tell me to restore and use DateRangePicker with the prop withModal. 
Thanks!